examples/user_guide/Plotting_with_Bokeh.ipynb

```diff
@@ -244,15 +244,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Overlay [show_legend=False height=400 width=600] ErrorBars (line_width=5) Scatter(alpha=0.2 size=6)\n",
+    "%%opts Overlay [show_legend=False height=400 width=600] ErrorBars (line_width=5) Scatter [jitter=0.2] (alpha=0.2 size=6)\n",
```

Comment:
Just checking that ``jitter`` complains if it is used without categorical axes (it should)....

Reply:
Opened an issue here: https://github.com/ioam/holoviews/issues/2236